app/src/main/java/com/project/technion/appark/fragments/ViewAllOffersFragment.java

```diff
@@ -30,16 +39,18 @@
 
 public class ViewAllOffersFragment extends Fragment {
 
-    private static final String  TAG = "ViewAllOffersFragment";
+    private static final String TAG = "ViewAllOffersFragment";
     private ListView mListView;
     private OffersAdapter mAdapter;
 
     private DatabaseReference mDatabaseReference;
     private FirebaseUser mUser;
     private FirebaseAuth mAuth;
     private View mRootView;
-    private static int lastSortIndex = -1;
+    private static SortingBy lastSortMethod = SortingBy.PRICE_LOWEST; // default sorting method
```

Comment:
I think it should be by lowest distance by default
```suggestion
    private static SortingBy lastSortMethod = SortingBy.DISTANCE_LOWEST; // default sorting method
```